### ISSUE TITLE
build failure

### ISSUE THREAD

**Issue by adarapata** [gs2io/gs2-sdk-for-unity#112]
Hello.

Follow the steps from the following page
If I install gs2-sdk-fot-unity, an error occurs during build.

https://app.gs2.io/docs/index.html#download

```
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Account\ScriptableObject\Namespace.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Enhance\ScriptableObject\Rate.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Enhance\ScriptableObject\Material.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Version\ScriptableObject\CurrentVersion.cs(21,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Exchange\ScriptableObject\Rate.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Exchange\ScriptableObject\Namespace.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Experience\ScriptableObject\Namespace.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Experience\ScriptableObject\Experience.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Formation\ScriptableObject\PropertyForm.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Formation\ScriptableObject\Namespace.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Formation\ScriptableObject\Mold.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Gateway\ScriptableObject\Namespace.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Friend\ScriptableObject\Namespace.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Inbox\ScriptableObject\Namespace.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Inbox\ScriptableObject\Message.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Stamina\ScriptableObject\Stamina.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Stamina\ScriptableObject\Namespace.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Inventory\ScriptableObject\Namespace.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Inventory\ScriptableObject\ItemSet.cs(18,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Inventory\ScriptableObject\Item.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Inventory\ScriptableObject\Inventory.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Key\ScriptableObject\Namespace.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Key\ScriptableObject\Key.cs(19,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Showcase\ScriptableObject\Showcase.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Limit\ScriptableObject\Namespace.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Showcase\ScriptableObject\Namespace.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Limit\ScriptableObject\Limit.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Showcase\ScriptableObject\DisplayItem.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Mission\ScriptableObject\Namespace.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Mission\ScriptableObject\MissionTask.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Mission\ScriptableObject\MissionGroup.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Mission\ScriptableObject\MissionCounter.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Money\ScriptableObject\Wallet.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Money\ScriptableObject\Namespace.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Quest\ScriptableObject\QuestGroup.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Quest\ScriptableObject\Quest.cs(17,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Library\PackageCache\io.gs2.unity.sdk@2022.7.1\Gs2Quest\ScriptableObject\Namespace.cs(16,17): error CS0103: The name 'AssetDatabase' does not exist in the current context
Error building Player because scripts had compiler errors
```

The `AssetDatabase` is only available on UnityEditor, so there is an error at build time.

It appears to have changed from the following commit
https://github.com/gs2io/gs2-sdk-for-unity/commit/3f85e3ffb7620bd689cda7431b461e16bb90e192

Unity Version 2021.2.8f
gs2-sdk-for-unity 2022.7.1

**Comment by kmiyamoto2021** [comment]
Hello, @adarapata 

Thank you for your report about the build error.
We have fixed the above bug in GS2 SDK for Unity 2022.7.2.

We appreciate your continued cooperation.

**Comment by adarapata** [comment]
Thanks for responding so quickly. 👍
